applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -72,10 +77,15 @@ The possible cases that impact how the host to which the nRF desktop dongle is c
 When reports are forwarded to the :ref:`nrf_desktop_usb_state`, each HID-class USB device is treated independently.
 This means that the |hid_forward| can concurrently send reports to every available HID-class USB device, if a report from the linked peripheral is available.
 
+HID boot protocol reports are received with report ID equal :c:enum:`REPORT_ID_RESERVED` and it is later changed based on  :c:struct:`bt_hogp` objects by |hid_forward| to either :c:enum:`REPORT_ID_BOOT_KEYBOARD` or :c:enum:`REPORT_ID_BOOT_MOUSE`.
+While receiving subscription event, |hid_forward| updates HID protocol of Bluetooth connected peripherals that are associated with given subscriber.
+
 Forwarding HID input reports
 ============================
 
 After :ref:`nrf_desktop_ble_discovery` successfully discovers a connected peripheral, the |hid_forward| automatically subscribes for every HID input report provided by the peripheral.
+Subscriber can use either HID boot protocol or HID report protocol, both protocols cannot be used at the same time.
+With current implementation, the :ref:`nrf_desktop_usb_state` supports either HID boot keyboard or HID boot mouse reports, because the HID boot protocol code is set using a single Kconfig option that is common for all of the instances of USB HID.
```

Comment:
```suggestion
In the current implementation, the :ref:`nrf_desktop_usb_state` supports either the HID boot keyboard or the HID boot mouse reports, because the HID boot protocol code is set using a single Kconfig option that is common for all of the instances of the USB HID.
```